clients/simple/financials.py

```diff
@@ -0,0 +1,46 @@
+# COGS
+
+# Inputs
+production_volume = 500
+prompt_tokens_per_request = 500  # Server logs print the number of prompt and completion tokens used on each request
+completion_tokens_per_request = 30
+# Token prices from https://openai.com/pricing for gpt-3.5-turbo-0125
+prompt_token_price = 0.0000005  # $/token
+completion_token_price = 0.0000015  # $/token
+
+stt_runtime = 0.15  # seconds
+tts_runtime = 2.0  # seconds per inference
+ec2_price = 0.526  # $/hour
```

Comment:
can you add some justification of these estimates in the comments?

- what machine has that `ec2_price`?
- tts_runtime on what machine, what model, approximately how long of a sentence?
- stt_runtime on what machine, what model ... anything else relevant here?

Reply:
ballpark is fine, just for the sake of documenting your work so we don't forget by the time return to it.